config/params.php

```diff
@@ -23,6 +23,10 @@
     'LiDAR' => "http://www.phenome-fppn.fr/vocabulary/2017#LiDAR",
     'Spectrometer' => "http://www.phenome-fppn.fr/vocabulary/2017#Spectrometer",
     'Installation' => "http://www.phenome-fppn.fr/vocabulary/2017#Installation",
+    'UAV' => 'http://www.phenome-fppn.fr/vocabulary/2017#UAV',
+    'UAVDocument' => 'http://www.phenome-fppn.fr/vocabulary/2017#AcquisitionSessionUAVDocument',
```

Comment:
Please use the same name AcquisitionSessionUAVDocument for the key

Reply:
Done